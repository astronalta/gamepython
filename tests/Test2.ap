class Test2 {


    foo() {
    }

}
