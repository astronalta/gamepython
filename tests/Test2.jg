# jgtest print_ast on

Test2 < Object {

    foo() {
    }

}
