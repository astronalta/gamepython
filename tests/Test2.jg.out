Module
  name: 
  feature0: Class
    name: Test2
    comment:
    feature0: Function
      name: foo
      type: Void
      comment:
      formal0: Formal
        name: __self
        type: Test2
      block: Block
    feature1: Function
      name: @destroy
      type: Void
      formal0: Formal
        name: __self
        type: Test2
      block: Block
    feature2: Function
      name: @init
      type: Void
      block: Block
